src/test/resources/org/jenkinsci/plugins/ewm/casc/obsolete-configuration-as-code.yaml

```diff
@@ -0,0 +1,16 @@
+jenkins:
+  systemMessage: "Jenkins configured automatically by Jenkins Configuration as Code Plugin\n\n"
```

Comment:
This seems unused

Reply:
The file itself or the systemMessage line?

Reply:
Context implies the file itself, I will remove it from my PR to @imaffe branch. :)